docs/windows.md

```diff
@@ -32,4 +32,8 @@ If the Grafana Agent is installed using the installer, it can be uninstalled via
 
 ## Logs
 
-When Grafana Agent is running as a Windows Service the Grafana Agent will write logs to Windows Event Logs. When running as executable, Grafana Agent will write to standard out. The logs will be written with the source of `Grafana Agent`.
\ No newline at end of file
+When Grafana Agent runs as a Windows Service, the Grafana Agent will write logs to Windows Event Logs. When running as executable, Grafana Agent will write to standard out. The logs will be written with the source of `Grafana Agent`.
```

Comment:
Can you add the eventlog name we use here? That'd make it easier for people to set up the Agent to collect logs from its own service.